apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -418,6 +418,11 @@ type FailedResourcePlacement struct {
 type ClusterResourcePlacementConditionType string
 
 const (
+	// ClusterResourcePlacementValidResourceSelectorsConditionType indicates whether cluster resource selectors have been
+	// recognized as valid.
+	// This will be false if the resource is not a cluster scope resource, or it's a reserved namespace or GVR is invalid.
+	ClusterResourcePlacementValidResourceSelectorsConditionType ClusterResourcePlacementConditionType = "ValidResourceSelectors"
```

Comment:
we will have a webhook to validate CRP, why do we need a condition for this?

Reply:
webhook may catch most of errors but it still could happen that, webhook validates the selector and then CRD is deleted during the runtime and gvk becomes invalid when crp controller validates.